test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 codecov
 pytest-cov
+google-compute-engine <= 2.7.999
```

Comment:
What is this used for?

Reply:
problem in the boto library, workaround for that - https://github.com/RaRe-Technologies/gensim/issues/898